util/cell2matmsspoly.m

```diff
@@ -0,0 +1,7 @@
+function output = cell2matmsspoly(C)
+output = C{1};
+for i = 2:1:length(C)
+   output = [output C{i}]; 
```

Comment:
Can you figure out the size of output beforehand? Dynamically growing the size of output will result in a lot of copies --> slow code.